ui/mirage/scenarios/default.js

```diff
@@ -13,6 +13,7 @@ const allScenarios = {
   allNodeTypes,
   everyFeature,
   emptyCluster,
+  allocationFileExplorer, // FIXME for stable preview links only
```

Comment:
I have no objections to short-lived custom scenarios, but I'm curious what this bought you that the existing scenarios didn't, just for future tech debt plans. Was it consistent IDs?

Reply:
exactly, I like being able to share a deep link that I know will work 😀